NOTICE

```diff
@@ -10,22 +10,10 @@ Copyright Red Hat, Inc. and/or its affiliates.. All Rights Reserved.
 
 This product also includes the following third-party components:
 
-* Twitter bootstrap version: 2.3.2
-  Downloaded from: https://getbootstrap.com/
-  License: Apache License v2.0 (just like OptaPlanner)
-
-* jQuery version: 1.9.1
-  Downloaded from: http://jquery.com/
-  License: MIT (which can be used by Apache License v2.0 projects)
-
 * google-code-pretty version: 4-Mar-2013
   Downloaded from: http://code.google.com/p/google-code-prettify
   License: Apache License v2.0 (just like OptaPlanner)
 
-* jQuery version: 3.3.1
-  Downloaded from: http://jquery.com/
-  License: MIT - this is compatible with ASL 2.0: http://www.apache.org/legal/resolved.html#category-a
-
 * jstree version: 3.3.5
   Downloaded from: https://www.jstree.com/
   License: MIT - this is compatible with ASL 2.0: http://www.apache.org/legal/resolved.html#category-a
```

Comment:
Thank you for the great work, @jomarko !

As mentioned in https://github.com/apache/incubator-kie-issues/issues/1616#issue-2656082830 , `NOTICE` file only contains contents of `NOTICE` files of the third-party components.

I confirmed that the repositories below don't have `NOTICE` files, so we can simply remove these items from our `NOTICE` file.

- https://github.com/vakata/jstree/
- https://github.com/googlearchive/code-prettify
- https://github.com/twbs/bootstrap/
- https://github.com/jquery/jquery

Reply:
Also please add jstree to `LICENSE` file. (If `optaplanner-docs` is excluded from source distribution using the release CI, we don't even need to add this to `LICENSE`)

```
optaplanner-docs/src/modules/ROOT/images/website/jstree/32px.png
optaplanner-docs/src/modules/ROOT/images/website/jstree/40px.png
optaplanner-docs/src/modules/ROOT/images/website/jstree/jstree.js
optaplanner-docs/src/modules/ROOT/images/website/jstree/jstree.min.js
optaplanner-docs/src/modules/ROOT/images/website/jstree/style.css
optaplanner-docs/src/modules/ROOT/images/website/jstree/throbber.gif
```

Reply:
`optaplanner-docs` is included in the source distribution. I will add corresponding entry into `LICENSE` for the `jstree`